rackspace/src/main/java/org/jclouds/examples/rackspace/cloudfiles/GenerateTempURL.java

```diff
@@ -105,6 +109,19 @@ private void createContainer() throws IOException {
       }
    }
 
+   private void updatePrivateURLKey() throws IOException {
+      // The key can be changed to quickly expire all existing signed URLs.
+      AccountApi accountApi = blobStore
+            .getContext()
```

Comment:
Yes, the example seems confusing now that you mentioned that. Personal opinion: it would make more sense to completely remove the blobStoreContext variable and use getContext() instead, just so users have an easier time reading the code (it's a short example).